.github/workflows/integration-test.yaml

```diff
@@ -14,25 +14,39 @@ jobs:
   check:
     runs-on: macos-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          - platform: iOS
+            lane: test_iOS16_2
+
+          - platform: tvOS
+            lane: test_tvOS16_1
+
+          - platform: macOS
+            lane: test_macOS
+
     env:
       LC_CTYPE: en_US.UTF-8
       LANG: en_US.UTF-8
       ABLY_ENV: sandbox
 
     steps:
-      - name: Check out SDK repo
+      - name: Checkout repo
         uses: actions/checkout@v2
         
-      - name: Select Specific Xcode Version (14.2)
+      - name: Select Xcode (14.2)
         run: |
           sudo xcode-select -s /Applications/Xcode_14.2.app
           echo "Selected Xcode version:"
           xcodebuild -version
 
-      - name: Log environment information
+      - name: Environment Info
         run: ./Scripts/log-environment-information.sh
 
-      - name: Check out xcparse repo
+      - name: Checkout xcparse repo
```

Comment:
"Check out" is the correct wording here (see e.g. the wording in `man git-checkout`). Ditto above.

Reply:
https://github.com/ably/ably-cocoa/pull/1840/commits/b9e17f004d54bd851c950e953d278127744e4728